spec/artifact-upload.spec.js

```diff
@@ -0,0 +1,91 @@
+"use strict";
+var config, container, cryptoAsync, fileCreator, fsAsync, logger;
+
+container = require("./helpers/lib/container")();
+logger = container.resolve("logger");
+fsAsync = container.resolve("fsAsync");
+cryptoAsync = container.resolve("cryptoAsync");
+fileCreator = container.resolve("fileCreator");
+config = container.resolve("config");
+
+
+/**
+ * Figures out the md5 hash of a local file.
+ *
+ * @param {string} file
+ * @return {Promise.<string>}
+ */
+function loadLocalMd5Hash(file) {
+    var hash, promise, stream;
+
+    stream = fsAsync.createReadStream(file);
+    hash = cryptoAsync.createHash("md5");
+
+    promise = new Promise((resolve, reject) => {
```

Comment:
No benefit. My guess is I was using the variable for something at one point. Fixed